### DISCUSSION TITLE
CSS PRACTICA (ESQUEMA)

### DISCUSSION THREAD

**Discussion by BrunoRamirezG** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#416] [Q&A]
Holaa!
Veo que en CSS ya nadamas aparecen el código en los links, eso es todo? 
Digo, esque en los anteriores links habia textos para leer.

**Comment by OscarPedroza** [comment]
Buenas noches, la explicación de ese código se encuentra en los diferentes lives que se realizaron en la semana 3

**Reply by BrunoRamirezG** [reply]
¡Muchas Gracias!

**Comment by doguedogue** [comment]
Asi es, está el código de ejemplo, los programas y las prácticas.

Si quieres lectura hay mucha sobre CSS en la red:
- https://developer.mozilla.org/es/docs/Web/CSS
- https://www.w3schools.com/css/

**Reply by BrunoRamirezG** [reply]
¡Muchas gracias!
